cli/CHANGELOG.md

```
@@ -70,9 +132,9 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 ## [8.3.4](https://github.com/Cap-go/capacitor-plus/compare/8.3.3...8.3.4) (2026-05-07)
 
 
-### Bug Fixes
+## Bug Fixes
```

markdownlint-cli2 (0.23.2)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)